### PR TITLE
fix: updated telemetry

### PR DESCRIPTION
**Pull request by jnikhila**
## Checklist
I have:
- [x] run the content through Grammarly
- [ ] linked to sample apps when relevant
- [x] added the meta description for each page in the PR
- [ ] minimized the callouts and added only when necessary
- [ ] added the `queryString` parameter to the Tabs (if used)
- [ ] masked PII in images. For example, login credentials, account details, and more
- [ ] added images only when necessary
- [ ] deleted the images that are no longer used for the updated pages in the PR
- [ ] followed the image file naming convention while renaming or adding new images. (Use lowercase letters, dashes between words, and be as descriptive as possible)
- [ ] used the `<figure/>` tag instead of a markdown representation for images
- [ ] added the `<figcaption/>` tag to add a caption to the image
- [ ] added the `alt` attribute in the `` tag
- [ ] verified and updated the cross-references or created redirect rules for the changed or removed content 
- [ ] reviewed and applied the style changes for UI formatting. For example, Bold the UI elements(Buttons on screen) used in the doc.

### CODE REVIEW

File: website/docs/product/telemetry.md [hunk 1]
```diff
@@ -8,58 +12,102 @@ Appsmith is a lightweight proxy and **does not capture** any data returned by yo
...
-  "userId": "70280e5dd9e61e5e91526ac8704bbd68d3f75ebad67ba439f4c354d7",
 }
+
 ```
 
-### Navigation and clicks
+### Usage pulse for billing
 
-The client captures anonymous behavioral data around navigation and clicks.
+The Appsmith client triggers a usage pulse whenever a user performs an action on their Appsmith instance. The usage pulse primarily contains information such as the timestamp, a hashed `userId` to map the user, and the app mode in which the action was performed. The data is collected on the server-side and sent to the Appsmith cloud services in batches every hour irrespective of whether telemetry is turned on or off. These pulses are then processed to present aggregate information to customers on customer portal. This data is collected irrespective of whether telemetry is turned on or off.
```
Comment (pranayagarwal96) [inline website/docs/product/telemetry.md:88]: I'd just add one more point here at the start or end - "This data is collected only for all paying customers"

Comment (jnikhila) [reply website/docs/product/telemetry.md]: done.

**Review by pranayagarwal96 — DISMISSED** [review]

**Review by pranayagarwal96 — APPROVED** [review]

File: website/docs/product/telemetry.md [hunk 2]
```diff
@@ -8,58 +12,102 @@ Appsmith is a lightweight proxy and **does not capture** any data returned by yo
 
 ### Keep-alive ping
 
-The Appsmith server sends a keep-alive ping every 6 hours to indicate that it's still running without any errors & captures anonymous usage data around which plugins are being used.
+The Appsmith server sends a keep-alive ping every 6 hours to indicate that it's still running without any errors & captures anonymous usage data around which plugins are being used. This data is collected irrespective of whether telemetry is turned on or off.
```
Comment (pranayagarwal96) [inline website/docs/product/telemetry.md:15]: This ping is every 2 hours today, actually I had updated that in the Notion doc but missed checking it here. Also can we just say "The Appsmith server sends a keep-alive ping every 2 hours to check it instance is still running without any errors", we don't need any of the text written after that

File: website/docs/product/telemetry.md [hunk 3]
```diff
@@ -8,58 +12,102 @@ Appsmith is a lightweight proxy and **does not capture** any data returned by yo
...
+        "segment_source_name": "cloud-services",
+        "type": "fetch-release-notes",
+        "version": "v1.9.15"
+    }
+}
+```
 
-**Sample event**
+### Server setup ping
+Appsmith server sends a ping first time a new instance is created. This data is collected irrespective of telemetry is turned on or off.
```
Comment (pranayagarwal96) [inline website/docs/product/telemetry.md:45]: "This data is collected irrespective of **whether** telemetry is turned on or off (just trying to be consistent with the language above)

File: website/docs/product/telemetry.md [hunk 4]
```diff
@@ -8,58 +12,102 @@ Appsmith is a lightweight proxy and **does not capture** any data returned by yo
...
-  "userId": "70280e5dd9e61e5e91526ac8704bbd68d3f75ebad67ba439f4c354d7",
 }
+
 ```
 
-### Navigation and clicks
+### Usage pulse for billing
 
-The client captures anonymous behavioral data around navigation and clicks.
+The Appsmith client triggers a usage pulse whenever a user performs an action on their Appsmith instance. The usage pulse primarily contains information such as the timestamp, a hashed `userId` to map the user, and the app mode in which the action was performed. The data is collected on the server-side and sent to the Appsmith cloud services in batches every hour irrespective of whether telemetry is turned on or off. These pulses are then processed to present aggregate information to customers on customer portal. This data is collected only for paying customers, regardless of whether telemetry is on or off.
```
Comment (pranayagarwal96) [inline website/docs/product/telemetry.md:88]: "regardless of whether telemetry is on or off" is repeated twice, can we remove one of them?

File: website/docs/product/telemetry.md [hunk 5]
```diff
@@ -91,52 +139,72 @@ The client captures anonymous behavioral data around navigation and clicks.
...
+    "type": "track",
+    "userId": "70280e5dd9e61e5e91526ac8704bbd68d3f75ebad67ba439f4c354d7",
 }
-
 ```
 
 ## Disable telemetry
 
-You can either disable the telemetry from [**Admin Settings**](telemetry.md#admin-settings) or by making changes to the [**`docker.env`**](telemetry.md#docker.env-file) file.
+Telemetry sharing is an optional setting, and you can disable telemetry either from Admin Settings or by making changes to the environment variable.
```
Comment (pranayagarwal96) [inline website/docs/product/telemetry.md:188]: Can we just say "Sharing telemetry is optional, and you can disable telemetry..."?

**Review by pranayagarwal96 — COMMENTED** [review]
Few minor suggestions on language across the doc

File: website/docs/product/telemetry.md [hunk 6]
```diff
@@ -1,65 +1,113 @@
...
+---
+
 # Telemetry
 
 Telemetry in Appsmith refers to the collection of data about how users interact with the platform. This data helps the Appsmith team identify usage patterns, troubleshoot issues, and make informed decisions about new features and improvements.
 
 ## Data collected by Appsmith
 
-Appsmith is a lightweight proxy and **does not capture** any data returned by your APIs, databases, or third-party tools. All data captured from a self-hosted instance is completely anonymised.
+Appsmith is a lightweight proxy and **does not capture** any data returned by your APIs, databases, or third-party tools. All data captured from a self-hosted instance is completely anonymised for user information.
```
Comment (pranayagarwal96) [inline website/docs/product/telemetry.md:11]: "Any user data captured from a self-hosted instance is completely anonymised"

Comment (jnikhila) [reply website/docs/product/telemetry.md:11]: "All data captured from a self-hosted instance is completely anonymised for user information" explicitly emphasizes that the anonymization is specifically for user information. Let's keep it as is.

File: website/docs/product/telemetry.md (hunk 2 above)
Comment (jnikhila) [reply website/docs/product/telemetry.md:15]: Will rewrite this.

File: website/docs/product/telemetry.md (hunk 3 above)
Comment (jnikhila) [reply website/docs/product/telemetry.md:45]: good catch

File: website/docs/product/telemetry.md (hunk 2 above)
Comment (jnikhila) [reply website/docs/product/telemetry.md:15]: Though there is no checking here - the server is sending a ping to indicate it's alive.

File: website/docs/product/telemetry.md [hunk 7]
```diff
@@ -1,65 +1,113 @@
...
 
 ### Keep-alive ping
 
-The Appsmith server sends a keep-alive ping every 6 hours to indicate that it's still running without any errors & captures anonymous usage data around which plugins are being used.
+The Appsmith server sends a keep-alive ping every 2 hours to indicate that it's still running without any errors. Additionally, it captures anonymous usage data to track the usage patterns of plugins. This data is collected irrespective of whether telemetry is turned on or off.
```
Comment (pranayagarwal96) [inline website/docs/product/telemetry.md:15]: "Additionally, it captures anonymous usage data to track the usage patterns of plugins" - why do we need this it "Keep-alive ping" section, it has nothing to do with it. I'm saying lets just get rid of it

**Review by pranayagarwal96 — APPROVED** [review]

**Review by danciaclara — APPROVED** [review]

**Pull request merged**